.github/workflows/minw64.yml

```diff
@@ -0,0 +1,59 @@
+name: MinGW64 Windows Build
+
+on:
+  push:
+    branches:
+      - master
+      - release-**
+    paths:
+    - 'src/**'
+    - 'external/**'
+    - 'python/**'
+    - 'tests/**'
+    - 'ms-windows/**'
+    - 'CMakeLists.txt'
+    - '.github/workflows/mingw64.yml'
+  pull_request:
+    branches:
+      - master
+      - release-**
+    paths:
+    - 'src/**'
+    - 'external/**'
+    - 'python/**'
+    - 'tests/**'
+    - 'ms-windows/**'
+    - 'CMakeLists.txt'
+    - '.github/workflows/mingw64.yml'
+
+
+jobs:
+  build:
+    name: MinGW64 Windows Build
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v1
+      - name: Build Docker Container with Build Environment
+        run: docker build -t qgis3-mingw-buildenv -f ms-windows/mingw/qgis3-build-deps-mingw.dockerfile .
+
+      - name: Build QGIS Application
+        run: docker run -w /QGIS -v $PWD:/QGIS qgis3-mingw-buildenv ms-windows/mingw/build.sh x86_64 nodebug 4
```

Comment:
can you add ccache similarly to what has been done in 4fc02733cb7bdf517eb5845f2883e04ef2e6f428